homework-16/index.ts

```diff
@@ -0,0 +1,20 @@
+import { customChecker} from "./custom.checker";
+
+//positive tests7
+customChecker(1, 2, 3, '+');
+customChecker(7, 2, 14, '*');
+customChecker(9, 3, 3, '/');
+customChecker(100, 70, 30, '-');
+customChecker(44, 44, 88, '+');
```

Comment:
Please use test design technics while writing your tests. For example it would be nice to add tests for adding `0` to a positive number, adding two negative numbers while testing addition method.